VAMobile/src/components/DefaultList.tsx

```diff
@@ -52,6 +53,7 @@ const DefaultList: FC<DefaultListProps> = ({
       titleA11yLabel={titleA11yLabel}
       rightTitleText={rightTitleText}
       rightTitleTextA11yLabel={rightTitleTextA11yLabel}
+      alertContent={alertContent}
```

Comment:
Why is this being added to default list? This is intended to just be lines of text